ApollosProject/apollos-prototype
# Packagizes the `LikeContentButtonConnected` component in `@apollosproject/ui-connected`

## DESCRIPTION
<img src="https://user-images.githubusercontent.com/2053547/72649341-64852800-394b-11ea-93dd-41a255b02b4c.gif" width="30%" />

### What does this PR do, or why is it needed?
This PR packages the  `LikeContentButtonConnected` component into `@apollosproject/ui-connected`.

### What design trade-offs/decisions were made?
N/A
### How do I test this PR?

1. open app
2. like content
3. check recently liked section in Connect tab
4. updated with your liked content? 💥 

### What automated tests did you write?
Same coverage as before.
## TODO:

- [x] Upload GIF(s) of iOS

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
